testng-core/src/test/kotlin/org/testng/internal/samples/classhelper/issue1339/BabyPandaSample.kt

```diff
@@ -0,0 +1,24 @@
+package org.testng.internal.samples.classhelper.issue1339
+
+class BabyPandaSample : LittlePandaSample() {
+    private var name: String = "Po"
+
+    override fun toString(): String {
+        return "BabyPanda{name='$name'}"
+    }
+
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+
+        other as BabyPandaSample
+
+        if (name != other.name) return false
+
+        return true
+    }
+
+    override fun hashCode(): Int {
```

Comment:
`override fun hashCode() = name.hashCode()`

Reply:
`hashCode()` and `toString()` I usually let the IDE generate it for me and don't fiddle around with it. Fixed